books/kestrel/apt/utilities/find-a-base-case.lisp

```diff
@@ -0,0 +1,154 @@
+;; Author: Grant Jurgensen (grant@kestrel.edu)
+
+(in-package "ACL2")
+
+(include-book "std/util/bstar" :dir :system)
+(include-book "kestrel/utilities/forms" :dir :system)
+(include-book "kestrel/terms-light/expr-calls-fn" :dir :system)
+(include-book "kestrel/utilities/translate" :dir :system)
+(include-book "kestrel/utilities/magic-macroexpand" :dir :system)
+(local (include-book "kestrel/typed-lists-light/pseudo-term-listp" :dir :system))
+(local (include-book "kestrel/typed-lists-light/symbol-listp" :dir :system))
+
+(defthm pseudo-termp-of-if-condition
```

Comment:
Can these three theorems be made local, so they do not affect the books that they include this tool directly or indirectly?